### ISSUE TITLE
Update period field for program requisitions

### ISSUE THREAD

**Issue by wlthomson** [openmsupply/mobile#1341]
## Description

Soliciting feedback for current UX for displaying periods in program requisitions.

### Context

Period info is a bit confusing to read.

## Design discussion

Periods are currently displayed in the following format:

```
periodName -- periodStartDate - periodEndDate
```

e.g.

```
Period -- 09/30/19 - 10/29/19
```

This is a bit confusing for me, too many dashes. Proposing a simple change to:

```
periodName (periodStartDate - periodEndDate)
```

e.g.

```
Period (09/30/19 - 10/29/19)
```

### Architecture

N/A.

**Comment by joshxg** [comment]
ok

**Comment by wlthomson** [comment]
Fixed by #1341.
